### PR TITLE
Add kibana-push command

### LINKED ISSUE #36: [FR] Automatically create Kibana branch/PR
**Is your feature request related to a problem? Please describe.**
Definitely. DRY!

**Describe the solution you'd like**
A simple command to `python -m detection_rules` or `make` that will generate the Kibana PR for us. Then we don't have to keep doing it.

**Describe alternatives you've considered**
N/A

**Additional context**
Related PR for 7.7 for reference https://github.com/elastic/kibana/pull/61903

### PR DESCRIPTION
**Pull request by rw-access**
## Issues
Closes #36
Depends on #37

## Summary
Wipes the directory in Kibana away and rebuilds with the latest version of the repository. Assumes you've already built the release, and will pull it from the filesystem. I think it's good to keep these decoupled, makes things one step easier. Exit codes should be enough if we want to automate multiple commands in a row.

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: detection_rules/main.py [hunk 1]
```diff
@@ -345,3 +347,58 @@ def test_rules(ctx):
 
     clear_caches()
     ctx.exit(pytest.main(["-v"]))
+
+
+@root.command("kibana-push")
+@click.pass_context
+@click.argument("local-repo", default=get_path("..", "kibana"))
+@click.option("--kibana-directory", "-d", help="Directory to overwrite in Kibana",
```
Comment (brokensound77) [inline detection_rules/main.py:355]: ++

File: detection_rules/main.py [hunk 2]
```diff
@@ -345,3 +347,58 @@ def test_rules(ctx):
...
+                ctx.exit(1)
+
+            url = f"git@github.com:{github_repo}.git" if ssh else f"https://github.com/{github_repo}.git"
+            subprocess.check_call([git_exe, "clone", url, local_repo, "--depth", 1])
+
+        def git(*args, show_output=False):
+            method = subprocess.call if show_output else subprocess.check_output
+            return method([git_exe, "-C", local_repo] + list(args), encoding="utf-8")
+
+        git("checkout", "master")
```
Comment (brokensound77) [inline detection_rules/main.py:389]: should we make `branch` an optional arg too, just in case?

File: detection_rules/main.py [hunk 3]
```diff
@@ -345,3 +347,58 @@ def test_rules(ctx):
...
+        git("rm", "-r", kibana_directory)
+
+        shutil.copytree(os.path.join(release_dir, "rules"), os.path.join(local_repo, kibana_directory))
+        git("add", kibana_directory)
+
+        git("commit", "-S", "-m", message)
+        git("status", show_output=True)
+
+        click.echo(f"Kibana repository {local_repo} prepped. Push changes when ready")
+        click.secho(f"cd {local_repo}", bold=True)
```
Comment (brokensound77) [inline detection_rules/main.py:401]: Did you mean for this to print or execute?

**Comment by brokensound77** [comment]
How would you feel about adding a `make` command as well for this @rw-access

File: detection_rules/main.py (hunk 3 above)
Comment (rw-access) [reply detection_rules/main.py:401]: Yeah, my thought was that it makes copy and paste easier so you can go to that dir easier to run git commands before pushing

**Pull request merged**